### DISCUSSION TITLE
Huge gap between lualine and tmux status line

### DISCUSSION THREAD

**Discussion by PCesteban** [LunarVim/LunarVim#3186] [Q&A] [answered]
Is there a way to reduce this gap? or even get rid of it?
![Screenshot from 2022-10-08 19-26-34](https://user-images.githubusercontent.com/48070832/194732229-3ebc854c-bc17-4b93-ae7e-eae635ffcdad.png)

**Comment by mateusabelli** [comment, marked as answer]
Hello @PCesteban could you tell me what's the output of `:set cmdheight`?

**Reply by PCesteban** [reply]
Hello @mateusabelli, the output is `cmdheight=2`.

Edit: Hey @mateusabelli, thanks I just set the value to 1 and the gap was reduced.

**Reply by mateusabelli** [reply]
Thats great! I asked first because I had a similar problem that was not cmdheight, to answer your last question you can set the value of cmdheight to 0 to hide it, its an experimental value in neovim 0.8

**Comment by totordev** [comment]
Hello, I just have a question for @PCesteban that is kind of unrelated but, what are you using to customise the tmux status line? Its looks great!

**Reply by PCesteban** [reply]
Hi @shebangcode, it's just a Nord theme plugin for tmux.
